linelist_investigations/report_sources/aaa_clean_linelist_2019-10-18.Rmd

```diff
@@ -373,8 +373,14 @@ This section defines age classes - see code for details.
 ```{r clean_age}
 
```

Comment:
Explain briefly what the code does (what problem / cases it covers) before the code chunk in plain English.